### PR TITLE
[Added] Radio module.

### PR DESCRIPTION
**Pull request by RicardoNeves**

### CODE REVIEW

File: src/app/radio/shared/components/radio-group/radio-group.component.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    {
+      multi: true,
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => RadioGroupComponent),
+    },
+  ],
+  selector: 'keds-radio-group',
+  templateUrl: './radio-group.component.html',
+})
+export class RadioGroupComponent implements AfterContentInit, ControlValueAccessor {
```
Comment (jmagalhaes-xing) [inline src/app/radio/shared/components/radio-group/radio-group.component.ts]: Are we using controls again? Why not `ngModel`?

File: src/app/radio/shared/components/radio/radio.component.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<label
+  [attr.for]="inputId"
+  class="keds-radio"
+  [class.keds-radio--checked]="checked"
+  [class.keds-radio--disabled]="isDisabled()"
+  (click)="handleClick($event)"
+>
+  <input
+    [(ngModel)]="checked"
```
Comment (jmagalhaes-xing) [inline src/app/radio/shared/components/radio/radio.component.html]: This should be in the end, next to `(ngModelChange)`

File: src/app/radio/shared/components/radio/radio.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+  value: string;
+
+  increment(): void {
+    this.clickCount += 1;
+  }
+}
+
+describe('RadioComponent', () => {
+
+  beforeEach(async(() => {
```
Comment (jmagalhaes-xing) [inline src/app/radio/shared/components/radio/radio.component.spec.ts]: `ng-bullet`

File: src/app/radio/shared/components/radio/radio.component.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  readonly uniqueId: string = `keds-radio-${nextUniqueId += 1}`;
+
+  @Input()
+  checked: boolean = false;
+
+  @Output()
+  click: EventEmitter<MouseEvent> = new EventEmitter<MouseEvent>();
+
+  @Output()
+  selected: EventEmitter<string> = new EventEmitter<string>();
```
Comment (jmagalhaes-xing) [inline src/app/radio/shared/components/radio/radio.component.ts]: Sort outputs

File: src/app/radio/shared/components/radio/radio.component.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  @Input()
+  required: boolean = false;
+
+  @Input()
+  tabIndex: number;
+
+  @Input()
+  value: string;
+
+  isDisabled(): boolean {
```
Comment (jmagalhaes-xing) [inline src/app/radio/shared/components/radio/radio.component.ts]: Sort

File: src/app/radio/radio.stories.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  }))
+  .add('Default', () => ({
+    template: `
+<h3>What's your Super Power?</h3>
+<keds-radio-group name="demo2">
+  <keds-radio value="flying">Flying</keds-radio>
+  <keds-radio value="mind control">Mind Control</keds-radio>
+  <keds-radio value="invisibility">Invisibility</keds-radio>
+  <keds-radio [disabled]="true" value="Other">I don't have one</keds-radio>
+</keds-radio-group>
```
[the remark this answers is not in the document]
Comment (RicardoNeves) [reply src/app/radio/radio.stories.ts]: Why?

File: src/app/radio/shared/components/radio-group/radio-group.component.ts (hunk 1 above)
Comment (RicardoNeves) [reply src/app/radio/shared/components/radio-group/radio-group.component.ts]: ngModel is available if you implement ControlValueAccessor

Comment (RicardoNeves) [reply src/app/radio/shared/components/radio-group/radio-group.component.ts]: plus controls

File: src/app/radio/radio.stories.ts (hunk 6 above)
[the remark this answers is not in the document]
Comment (RicardoNeves) [reply src/app/radio/radio.stories.ts]: for me looks like more code/files to handle when there's no need for that.

**Pull request merged**